### ISSUE TITLE
Allow multiple arrays to be passed into validate again

### ISSUE THREAD

**Issue by alexgarrett** [alexgarrett/violin#33]
Keep current way of passing into validate, but also allow for two arrays.

The reasoning for this is that, although the updated solution is cleaner, it doesn't allow for form rules to be defined anywhere else, e.g. within a RegisterForm class:

```php
class RegisterForm
{
	public static $rules = [
		'username' => 'required|min(3)|max(20)',
		'email' => 'required|email',
		'password' => 'required|min(6)',
		'password_confirm' => 'required|matches(password)'
	];
}
```

Not sure if this should be a separate method, or just check within `validate` if two arrays exist and modify the logic accordingly.

**Comment by oxyzero** [comment]
Fixed in #34.

**Comment by alexgarrett** [comment]
:+1:
